easybuild/easyconfigs/t/Tk/Tk-8.6.4-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.4'
+
```

Comment:
I agree that it could be broken but we don't really use it here so I don't know how to test and I wouldn't like to block this PR because of Tk. How about opening a separate issue for that?  

We probably need to debug setupext on other builds (since we don't get any log from the exceptions) to see if there's something particular happening here.